test/arrays/bharshbarg/popFrontLeak3.good

```diff
@@ -1,3 +1,6 @@
+popFrontLeak3.chpl:37: In function 'main':
+popFrontLeak3.chpl:42: warning: push_back is deprecated - please use List.extend
+popFrontLeak3.chpl:43: warning: pop_front is deprecated - please use List.pop
```

Comment:
I think it would be worth rewriting these popFrontLeak[1-3] tests to use the List type to ensure that the leak wasn't accidentally re-introduced.

Reply:
I've made an issue to remind myself to do this after `list` gets support for `init=` from an `_iteratorRecord`.